source/bson-binary-vector/bson-binary-vector.md

```diff
@@ -7,25 +7,77 @@ ______________________________________________________________________
 
 ## Abstract
 
-This document describes the subtype of the Binary BSON type used for efficient storage and retrieval of vectors. Vectors
-here refer to densely packed arrays of numbers, all of the same type.
+This document describes a new *Vector* subtype (9) for BSON Binary items, used to compactly represent ordered
```

Comment:
Would you please drop the 9 in this first sentence, or perhaps change to this?
"This document describes a new *Vector* subtype (9) of the BSON Binary type (5) ..."